sv_pipeline/genome/load_data.py

```diff
@@ -0,0 +1,181 @@
+import argparse
+import os
+import time
+import hail as hl
+import logging
+
+from hail_scripts.v02.utils.elasticsearch_client import ElasticsearchClient
+
+from sv_pipeline.utils.common import get_sample_subset, get_sample_remap, get_es_index_name, CHROM_TO_XPOS_OFFSET
+from sv_pipeline.genome.utils.mapping_gene_ids import load_gencode, get_gene_id
+
+logging.basicConfig(level=logging.INFO)
+logger = logging.getLogger(__name__)
+
+WGS_SAMPLE_TYPE = 'WGS'
+BASIC_FIELDS = {
+        'contig': lambda rows: rows.locus.contig.replace('^chr', ''),
+        'sc': lambda rows: rows.info.AC[0],
+        'sf': lambda rows: rows.info.AF[0],
+        'sn': lambda rows: rows.info.AN,
+        'svType': lambda rows: hl.if_else(rows.alleles[1].startswith('<INS:'), 'INS', rows.alleles[1].replace('[<>]', '')),
+        'start': lambda rows: rows.locus.position,
+        'end': lambda rows: rows.info.END,
+        'sv_callset_Hemi': lambda rows: rows.info.N_HET,
+        'sv_callset_Hom': lambda rows: rows.info.N_HOMALT,
+        'gnomad_svs_ID': lambda rows: rows.info.gnomAD_V2_SVID,
+        'gnomad_svs_AF': lambda rows: rows.info.gnomAD_V2_AF,
```

Comment:
Good point. We will discuss.

Reply:
I had done this for consistency with how we generally name things in seqr. We actually map `sc` and `sf` BACK to be called AC and AF when we parse the data from elasticsearch, but its easiest to export them with those names here for consistency with the exome SV pipeline schema.  However, since the gnomad fields are new in this data and don't need to match the exome SV fields I think it would be better to keep this as AF, especially since thats what its called in the VCF